manifestpy/manifestmap.py

```diff
@@ -0,0 +1,31 @@
+def mapManifestOfItems(readfile, wtype):
+    import os
+    from mapmain import mapMainManifest
+
+    path = os.path.dirname(os.path.abspath(__file__))
+    wfile = os.path.join(path, 'outputmain.json')
+    w = open(wfile, 'w+')
```

Comment:
For the `w` variable, I'm wondering if you can declare it closer to its first usage OR rename it to something more verbose.

If I'm understanding what you are doing you are building a JSON string. I'm thinking about how I might do this in Ruby. I would create an in-memory Hash object (or Dictionary) then use the JSON.dump method to write it to a string. I see in Python - https://stackoverflow.com/questions/17043860/python-dump-dict-to-json-file

My concern with the existing approach would be if the label had a `"` it could create a bad JSON object.